support/insights-operator/disabling-insights-operator.adoc

```diff
@@ -0,0 +1,23 @@
+[id="disabling-insights-operator"]
+= Disabling the Insights Operator
+include::modules/common-attributes.adoc[]
+:context: insights-operator-disabling
+
+toc::[]
+
+:FeatureName: The Insights Operator
+include::modules/technology-preview.adoc[leveloffset=+0]
+
+By default, the Insights Operator is enabled for {product-title}. To disable data collection by the Insights Operator, you must:
+
+. xref:../../support/insights-operator/disabling-insights-operator.adoc#insights-operator-new-pull-secret_insights-operator-disabling[Modify the global cluster pull secret] to disable Telemetry and the Insights Operator.
+. xref:../../support/insights-operator/disabling-insights-operator.adoc#images-update-global-pull-secret_insights-operator-disabling[Update the cluster] to use this modified pull secret.
+
+[WARNING]
+====
+It is not recommended to disable the Insights Operator, which also disables Telemetry. Be sure to review the xref:../../telemetry/opting-out-of-telemetry.adoc#telemetry-consequences-of-disabling-telemetry_telemetry[consequences of disabling Telemetry] before doing so.
```

Comment:
This would be the section that when collapsed together needs to describe all the cases - I think we can remove this bullet if we do.

Reply:
The comment on that page:

> However, Telemetry is embedded as a core component and serves as part of the overall function of the cluster. Therefore, opting out is strongly discouraged.

Is fairly weak.  I would reframe as:

> Remote Health Monitoring via Telemetry and the Insights Operator allows Red Hat to better detect and resolve errors that occur in production environments and to identify potential problems in pre-production systems before they can impact customer workloads.  Sharing health data with Red Hat allows better quality assurance during upgrades and helps Red Hat prioritize and triage issues.  In addition, enabling these components allows use of the Red Hat OpenShift Cluster Manager at `cloud.redhat.com` and simplifies subscription entitlement and support registration.